cypress/support/step-definitions/drawer-steps.js

```diff
@@ -0,0 +1,53 @@
+import {
+  drawerToggle,
+  drawerSidebar,
+  drawerSidebarContentInnerElement,
+} from '../../locators/drawer';
+
+When('I click on Drawer arrow {int} time(s)', (count) => {
+  for (let i = 0; i < count; i++) {
+    drawerToggle().first().click();
+  }
+});
+
+Then('sidebar should have class {word}', (className) => {
+  drawerSidebar().should('have.class', className);
+});
+
+Then('sidebar text is visible', () => {
+  drawerSidebarContentInnerElement(1).should('have.text', 'link a')
+    .and('be.visible');
+  drawerSidebarContentInnerElement(2).should('have.text', 'link b')
+    .and('be.visible');
+  drawerSidebarContentInnerElement(3).should('have.text', 'link c')
+    .and('be.visible');
+});
+
+Then('sidebar text is not visible', () => {
+  drawerSidebarContentInnerElement(1).should('have.text', 'link a')
+    .and('not.be.visible');
+  drawerSidebarContentInnerElement(2).should('have.text', 'link b')
+    .and('not.be.visible');
+  drawerSidebarContentInnerElement(3).should('have.text', 'link c')
+    .and('not.be.visible');
+});
+
+Then('toggle icon switched orientation to open', () => {
+  drawerToggle().should('have.css', 'transform', 'matrix(-1, 0, 0, 1, 0, 0)');
+});
+
+Then('toggle icon switched orientation to closed', () => {
+  drawerToggle().should('have.css', 'transform', 'none');
+});
+
+// When('change expandedWidth to {word}', (width) => {
```

Comment:
Can we use or remove this?

Reply:
removed, it was not needed as two steps below replaced that one.